transforms/csdl/transform.ps1

```diff
@@ -9,7 +9,9 @@ param (
     [string]
     $outputPath = "preprocess_csdl_test_output.xml",
     [bool]
-    $dbg = $false
+    $dbg = $false,
+    [bool]
+    $removeCapaAnnotations = $true
```

Comment:
can we replace Capa by Capability, I know it's a long name but Capa is not a common thing and I had to read the script to understand what this does

Reply:
Good call. I did that elsewhere, forgot to do that here.